build/messages/src/Messages.json

```diff
@@ -103,6 +115,10 @@
     "defaultMessage": "Red Hat Insights for SAP",
     "description": "Red Hat Insights for SAP"
   },
+  "insightsandsmartmanagement": {
+    "defaultMessage": "Insights and smart management",
```

Comment:
Should `smart management` be capitalized? It is in other places, so not sure of the casing.